ksql-cli/src/test/java/io/confluent/ksql/cli/console/ConsoleTest.java

```diff
@@ -68,42 +72,44 @@
 public class ConsoleTest {
```

Comment:
We should add a basicReadLine test that checks we return lines that don't contain cli-specific commands.

Reply:
Done.